xbmc/interfaces/Builtins.cpp

```diff
@@ -166,6 +167,7 @@ const BUILT_IN commands[] = {
   { "AlarmClock",                 true,   "Prompt for a length of time and start an alarm clock" },
   { "CancelAlarm",                true,   "Cancels an alarm" },
   { "Action",                     true,   "Executes an action for the active window (same as in keymap)" },
+  { "DialogOK",                   true,   "Shows an OK dialog on screen, specify header, then message." },
```

Comment:
my only concern with that, in #7204 we use `TextViewer` here we use `DialogOK`. One time we prefix it with dialog and one time we don't, one time we use a built-in name e.g. `Notification` which is related to the action, not the dialog which is the kaitoast dialog and one time we use the dialog name as built-in cmd. That's not really intuitive IMO. If we prefix the cmd with `Show` or `Dialog.` e.g. `ShowTextViewer` or `Dialog.TextViewer`, `ShowOK` or `Dialog.OK` this would indicates what these cmd's will do. Don't forget you are very familiar with our skin engine and you know each cmd and dialog.